root/release_group/set_cover_art.tt

```diff
@@ -26,33 +26,56 @@
           </div>
-          <p>
-            [%- descriptive_link(release) -%]<br />
-            [% release_dates(release.events) %]
-            [% release_countries(release.events) %]<br />
-            <span style="display: inline-block">[%- html_escape(release.combined_format_name) -%],</span>
-            <span style="display: inline-block">[%- l('Tracks:') %] [% release.combined_track_count -%]</span><br />
-            <span style="display: inline-block">[% release_label_list(release.labels) -%]
-              [%- IF release_label_list(release.labels) AND release_catno_list(release.labels) -%];
-              [%- END -%]
-              [% release_catno_list(release.labels) %]
-            </span>
-            <br />
-            [%- IF release.barcode -%]
-              [%- l('Barcode:') %] [% release.barcode -%]
+          <div class="release-description">
+            <p>
+              [%- link_entity(release) -%]<br />
+              [%- l('Artist:') %] [% artist_credit(release.artist_credit) -%]
+            </p>
+            [%- IF release.events -%]
+              <p>
+                [%- IF release_dates_list(release.events) -%]
+                  [%- l('Date:') %] [% release_dates_list(release.events) %]
+                   <br />
+                [%- END -%]
+                [%- IF release_countries_list(release.events) -%]
+                  [%- l('Country:') %] [% release_countries_list(release.events) %]
+                [%- END -%]
+              </p>
             [%- END -%]
-          </p>
-          [%- IF image -%]
             <p>
-              [%- comma_list(image.types) || '-' -%]<br />
-              [%- image.comment | html -%]
-              [%- IF originally_selected -%]
-                <strong>
-                  [% l('This is the current release group image') %]
-                </strong>
-              [%- END -%]
+            [%- l('Format:') %] [% html_escape(release.combined_format_name) -%]<br/>
+            [%- l('Tracks:') %] [% release.combined_track_count -%]
             </p>
+            [%- IF release.labels -%]
+              <p>
+                [%- IF release_label_list(release.labels) -%]
+                  [%- l('Label:') %] [% release_label_list(release.labels) -%]
+                  <br />
+                [%- END -%]
+                [%- IF release_catno_list(release.labels) -%]
+                  [%- l('Catalog#:') %] [% release_catno_list(release.labels) %]
+                [%- END -%]
+              </p>
+            [%- END -%]
+            [%- IF release.barcode -%]
+              <p>
+                [%- l('Barcode:') %] [% release.barcode -%]
+              </p>
+            [%- END -%]
+            [%- IF image -%]
+              <p>
+                [%- l('Image types:') %] [% comma_list(image.types) || '-' -%]<br />
+                [%- IF image.comment -%]
+                  [%- l('Image comment:') %] [%- image.comment | html -%]
+                [%- END -%]
```

Comment:
These lines can all be written using existing localizable messages instead:

```suggestion
                [%- add_colon(l('Cover Art')) -%]
                <ul>
                  <li>[%- l('Types:') %] [% comma_list(image.types) || '-' -%]</li>
                  [%- IF image.comment -%]
                    <li>[%- l('Comment:') %] [%- image.comment | html -%]</li>
                  [%- END -%]
                </ul>
```

@reosarevok: If you agree with that change, I would push it before updating beta.

Reply:
Approved on IRC, pushed this change to `master` via commit https://github.com/metabrainz/musicbrainz-server/commit/9f1d66950f7247f18fbc363a1189da27ace999da.